static-site/src/components/ListResources/Showcase.tsx

```diff
@@ -28,10 +34,15 @@ export const Showcase = ({cards, setSelectedFilterOptions}) => {
   )
 }
 
+type ShowcaseTileProps = {
+    data: Card
```

Comment:
<nitpick> `data` is pretty generic to start with; having it typed as `Card` increases the cognitive dissonance. Consider? 

```suggestion
    card: Card
```

Reply:
Good suggestion, I'll add this to deb9e0c22da15b8c05994e84abacd4917cd0f67c

Reply:
Added as part of e03c7b5f9a9bbbfaa82b816c03de00e17c5e73b1